lib/osf-components/addon/components/resources-list/template.hbs

```diff
@@ -14,7 +30,7 @@
         local-class='ResourceList'
 >
     <:item as |resource|>
-        <ResourcesList::ResourceCard @resource={{resource}} />
+        <ResourcesList::ResourceCard @resource={{resource}} @editable={{true}} />
```

Comment:
We'll need to check if `@registration.userHasWritePermission` to see if it should be editable.